test/blackbox-tests/test-cases/reason/jbuild

```diff
@@ -4,31 +4,31 @@
 (library
  ((name rlib)
   (public_name rlib)
-  (modules bar cppome foo hello pped)
+  (modules (bar cppome foo hello pped))
```

Comment:
Out of curiosity - why the change from `dune` to `jbuilder`?  Is Reason supported under Dune 1.0.0?

Reply:
Yeah, it's supported. I don't  quite remember as the PR was sitting around. I think it's because of a ppx reason. I'll try again.

Reply:
Thanks - just checking before I dive into testing dune against a local mixed OCaml/Reason codebase.